pkg/cli/admin/prune/deployments/data.go

```diff
@@ -76,7 +77,12 @@ func FilterDeploymentsPredicate(item metav1.Object) bool {
 	case *corev1.ReplicationController:
 		return len(appsutil.DeploymentConfigNameFor(v)) > 0
```

Comment:
I'm inclined to also apply here similar rule as below wrt using `OwnerRferences` since that's what garbage collector uses and it seems reasonable for the prune command to follow as well. This can be made as a followup.